css/index.css

```diff
@@ -112,9 +142,35 @@ p {
     justify-content: center;
 }
 
+.middle-content .boxes:after {
+    padding-bottom: 100%;
+    border: 1px solid red;
+}
+
+.middle-content .boxes .box:nth-of-type(2) {
+    background: forestgreen;
+    }
+
+.middle-content .boxes .box:nth-of-type(4) {
+    background: darkorchid;
+    }
+
+.middle-content .boxes .box:nth-of-type(6) {
+    background: hotpink;
+    }
+
+.middle-content .boxes .box:nth-of-type(8) {
+    background: indigo;
+    }
+
+.middle-content .boxes .box:nth-of-type(10) {
+    background: lawngreen;
+    }
```

Comment:
I am still impressed that you used the pseudo class `:nth-of-type`. As far as I know, no one else did this.